Documentation/Power.md

```diff
@@ -1,24 +1,44 @@
 # Power & Performance
 
-All of the irons are PWM controlled resistive heating elements.
+All of the irons are [PWM controlled](https://www.digikey.com/en/blog/pulse-width-modulation#) resistive heating elements.
 This means that the electronics in the handle can only turn the heating element on and off.
-This *means* that the power provided in the tip is 100% controlled by the supply voltage used.
+This *means* that the power provided in the tip is 100% controlled by the supply voltage used (higher voltage PSU = higher performance).
 
-Both irons at their simplest are just a resistor connected to your power source via a switch.
+Irons at their simplest are just a resistor (Ω) connected to your power source via a switch.
 
-When the switch is on, the power in the resistor is `Current (I) times Volts (V)`.
-Current through the resistor is `Volts (V) divided by Resistance (R)`.
-Combining these two gives the common equation, power is `Volts (V) squared / Resistance (R)`.
+- When the switch is on, the power in the resistor is: $P(watts) = V(volts) \times\ I(current=amps)$
+- Current through the resistor is:  $I(amps) = V(volts) ÷ Ω (resistance)$
+- Combining these gives some common equations for Power
+
+   $P(watts) = V(volts) * I(amps)$ or $P = V^2 ÷ Ω$
+
+The resistance of the tip is a fixed constant in ohms (Ω):
+- 6.2 Ω  Pine64 short tip
+- 8.0 Ω  TS100/Pinecil long tip
+- 4.5 Ω  TS80(P)
+
+This means the power delivered to the soldering tip is proportional to the voltage squared.
+Therefore the Pinecil and TS100 perform poorly when run off 12V power supplies and may issue a `Thermal Runaway` message (weak power supply).
+
+#### Use an [Ohm calculator](https://www.rapidtables.com/calc/electric/power-calculator.html#dc) to quickly derive watts.
+
```

Comment:
If we are going to do this table, can we also do the TS80P a tiny table with 9/12V values?